### PR TITLE
lxd/instance/lxc: Stop forkfile on restore

### PR DESCRIPTION
**Pull request by stgraber**
Signed-off-by: Stéphane Graber <stgraber@ubuntu.com>

### CODE REVIEW

File: lxd/instance/drivers/driver_lxc.go [hunk 1]
```diff
@@ -3495,6 +3495,10 @@ func (d *lxc) Restore(sourceContainer instance.Instance, stateful bool) error {
 
 	d.logger.Info("Restoring container", ctxMap)
 
+	// Wait for any file operations to complete.
+	// This required so we can actually unmount the container and restore its rootfs.
```
Comment (simondeziel) [inline lxd/instance/drivers/driver_lxc.go:3499]: `s/This required/This is required/` ?

File: lxd/instance/drivers/driver_lxc.go [hunk 2]
```diff
@@ -3649,7 +3653,7 @@ func (d *lxc) Delete(force bool) error {
 	}
 
 	// Wait for any file operations to complete.
-	// This is to required so we can actually unmount the container and delete it.
+	// This required so we can actually unmount the container and delete it.
```
Comment (simondeziel) [inline lxd/instance/drivers/driver_lxc.go:3656]: Here too, `is` sounds better to my ear.

**Review by tomponline — APPROVED** [review]
@stgraber shall we still merge this as it will be replaced by the "exclusive" lock mode we discussed earlier?

**Comment by stgraber** [comment]
@tomponline might as well, if anything, it will tell us where we need to use the exclusive mode :)

**Pull request merged**